interface/wx/webview.h

```diff
@@ -764,9 +768,32 @@ class wxWebView : public wxControl
             @NULL if it is not needed. This parameter is new since wxWidgets
             version 3.1.1.
         @return @true if there is a result, @false if there is an error.
+
+        @see RunScriptAsync()
```

Comment:
It would be nice to emphasize that async version should be preferred because of all the problems with the sync one.

Reply:
I was thinking the same, I'll at a notice recommending `RunScriptAsync()`